### DISCUSSION TITLE
Edit job interval after initialization

### DISCUSSION THREAD

**Discussion by federicoodorizzi** [breejs/bree#179] [Q&A] [answered]
Hey, is there a way to change the interval of a job while it is running? I was thinking of sending the new interval value using an API request, and then update the job accordingly. Is that possibile? Thanks!

**Comment by shadowgate15** [comment, marked as answer]
You would need to something like below:
```js
// stop job
bree.stop(NAME);

// find the job in config array
const idx = bree.config.jobs.findIndex(j => j.name === NAME);

// set with new interval
bree.config.jobs[idx] = { ...bree.config.jobs[idx], interval: NEW_INTERVAL };

// restart job
bree.start(NAME);
```

**Comment by neilenns** [comment]
Has anyone successfully got this to work? I tried today and kept getting in to situations where Bree just wouldn't run the job at all after changing the interval using the above code.
